### PR TITLE
Make "### Studies" a link and remove button from tooltip

### PR DESCRIPTION
**Pull request by dmfalke**
This was suggested at a recent map ux meeting. I think it makes sense.

image

### CODE REVIEW

File: packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx [hunk 1]
```diff
@@ -151,13 +147,20 @@ export function MapTypeHeaderStudyDetails(props: Props) {
           : outputEntity.displayNamePlural}
         {onShowStudies && studyEntity && (
           <>
-            {' '}
-            from {format(studyEntityCount.data.count)}{' '}
-            {studyEntityCount.data.count === 1
-              ? studyEntity.displayName
-              : studyEntity.displayNamePlural}
+            &nbsp;from&nbsp;
```
Comment (bobular) [inline packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx:150]:
```suggestion
            &nbsp;from
```
My OCD brain can't cope with the extra space before the link. Could we get rid of it please?

**Review by bobular — APPROVED** [review]
ALGTM!

File: packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx (hunk 1 above)
Comment (dmfalke) [reply packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx:150]: Without that, there would be no space between "from" and "### Studies"

Comment (bobular) [reply packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx:150]: There was for me when I tried it

Comment (dmfalke) [reply packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx:150]: I just tested again. Strange!

Comment (dmfalke) [reply packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx:150]: I'm seeing consistent behavior in all major browsers (on linux) :shrug:

**Pull request merged**